integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -174,36 +173,39 @@ def _create_transmissions(self, content_metadata_item_map, content_updated_mappi
                 integrated_channel_code=self.enterprise_configuration.channel_code(),
                 content_id=content_id,
                 deleted_at__isnull=False,
-            )
+            ).first()
```

Comment:
I would also find it useful to have the docstring for this method quickly explain what it does and the 3 cases it handles and what it does in each case, just briefly is fine

Reply:
mainly think it's useful now since the method is no longer as simple as it used to be, but more accurate!